testsuite/features/secondary/min_rhlike_openscap_audit.feature

```diff
@@ -38,11 +52,13 @@ Feature: OpenSCAP audit of Red Hat-like Salt minion
     When I follow "Audit" in the content area
     And I follow "xccdf_org.open-scap_testresult_standard"
     Then I should see a "Details of XCCDF Scan" text
-    And I should see a "RHEL-7" text
+    And I should see a "RHEL-8" text
     And I should see a "XCCDF Rule Results" text
-    When I enter "pass" as the filtered XCCDF result type
-    And I click on the filter button
-    Then I should see a "rpm_verify_permissions" link
+    # Workaround for the missing Rocky 8 openSCAP profile
```

Comment:
use WORKAROUND in upper case for ease of search

Reply:
Fixed.